docs/source/1.0/spec/aws/aws-json-1_0-protocol.rst

```diff
@@ -61,4 +61,208 @@ See
     This protocol only permits the :ref:`httpPayload-trait` to be applied to
     members that target structures, documents, strings, blobs, or unions.
 
-*TODO: Add specifications, protocol examples, etc.*
+
+----------------
+Supported traits
+----------------
+
+The ``awsJson1_0`` protocol supports the following traits
+that affect serialization:
+
+.. list-table::
+    :header-rows: 1
+    :widths: 20 80
+
+    * - Trait
+      - Description
+    * - :ref:`jsonName <jsonName-trait>`
+          - By default, the JSON property names used in serialized structures are
+            the same as a structure member name. The ``jsonName`` trait changes
+            the JSON property name to a custom value.
+
+
+----------------
+Protocol Headers
+----------------
+
+The ``awsJson1_0`` protocol uses the following headers:
+
+.. list-table::
+    :header-rows: 1
+    :widths: 20 20 80
+
+    * - Header
+      - Required
+      - Description
+    * - ``Content-Type``
+      - true
+      - This header has a static value of ``application/x-amz-json-1.0``.
+    * - ``Content-Length``
+      - true
+      - The standard ``Content-Length`` header defined by
+        `RFC 7230 Section 3.3.2`_.
+    * - ``X-Amz-Target``
+      - true for requests
+      - The value of this header is the :token:`shape name <identifier>` of the
+        service's :ref:`shape-id` joined to the
+        :token:`shape name <identifier>` of the operation's :ref:`shape-id`,
+        separated by a single period (``.``) character.
+
+        For example, the value for the operation ``ns.example#MyOp`` of the
+        service ``ns.example#MyService`` is ``MyService.MyOp``.
+
+
+-------------
+HTTP Bindings
+-------------
+
+Every request for the ``awsJson1_0`` protocol MUST be sent to the
+root URL (``/``) using the HTTP POST method.
+
+The ``awsJson1_0`` protocol does not support custom HTTP bindings.
+HTTP binding traits MUST be ignored if they are present.
```

Comment:
"HTTP binding traits" should probably link to that page of the spec so implementers can know.